spec/requests/internal/classified_listings_spec.rb

```diff
@@ -14,6 +14,10 @@
       put "/internal/listings/#{classified_listing.id}", params: {
         classified_listing: { title: "updated" }
       }
+      expect(ClassifiedListings::BustCacheWorker.jobs.size).to eq(1)
+
+      # executing the job to check if the method gets called
+      ClassifiedListings::BustCacheWorker.drain
```

Comment:
You can replace this with our helper: `sidekiq_perform_enqueued_jobs` for consistency, see https://github.com/thepracticaldev/dev.to/blob/master/spec/support/sidekiq_test_helpers.rb#L60

Reply:
sure fair point!